hr_expense_invoice/views/account_move_views.xml

```diff
@@ -1,22 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <record id="view_move_form" model="ir.ui.view">
         <field name="name">account.move.form - Add expense smart-button</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
             <div class="oe_button_box" position="inside">
-                <field name="expense_ids" invisible="1" />
+                <field invisible="1" name="expense_ids" />
```

Comment:
Name goes first

Same in similar cases, the previous version was correct.

Reply:
done